src/content/docs/opentelemetry/best-practices/opentelemetry-otlp.mdx

```diff
@@ -283,6 +283,14 @@ The mechanism to configure the endpoint will vary, but OpenTelemetry language SD
 
 New Relic uses [OTLP release v0.18.0](https://github.com/open-telemetry/opentelemetry-proto/releases/tag/v0.18.0). Later versions are supported but new features are not yet implemented. Experimental features which were are no longer available in 0.18.0 are not supported.
 
+## OTLP attribute types [#otlp-attribute-types]
+
+Attributes are a recurring concept in OpenTelemetry and OTLP. OpenTelemetry has a [standard attribute](https://opentelemetry.io/docs/specs/otel/common/#standard-attribute) definition, stating that attribute values are primitives (string, boolean, double floating point, 64-bit integer) or arrays of integers. However, at the OTLP protocol level, attributes are represented using a more expansive [`AnyValue`](https://github.com/open-telemetry/opentelemetry-proto/blob/4f69356d853029975649c3f38b06fc77d77975fc/opentelemetry/proto/common/v1/common.proto#L28-L40) definition. Because of this, it's possible for OTLP clients to send attributes which do not conform to OpenTelemetry standard definition.
```

Comment:
```suggestion
Attributes are a recurring concept in OpenTelemetry and OTLP. OpenTelemetry has a [standard attribute](https://opentelemetry.io/docs/specs/otel/common/#standard-attribute) definition, stating that attribute values are primitives (string, boolean, double floating point, 64-bit integer) or homogeneous arrays of primitives. However, at the OTLP protocol level, attributes are represented using a more expansive [`AnyValue`](https://github.com/open-telemetry/opentelemetry-proto/blob/4f69356d853029975649c3f38b06fc77d77975fc/opentelemetry/proto/common/v1/common.proto#L28-L40) definition. Because of this, it's possible for OTLP clients to send attributes which do not conform to OpenTelemetry standard definition.
```